zipline/finance/trading.py

```diff
@@ -128,7 +131,7 @@ def write_data(self, **kwargs):
 class SimulationParameters(object):
     def __init__(self, start_session, end_session,
                  trading_calendar,
-                 capital_base=10e3,
+                 capital_base=DEFAULT_CAPITAL_BASE,
```

Comment:
Not something to change right now, but I think eventually we'll get rid of this default entirely. I think of this class as internal now that users have `run_algorithm`.